src/config.ts

```diff
@@ -0,0 +1,10 @@
+export enum Source {
+  v1 = 'gcpv1_enriched.json',
+  v2 = 'gcpv2_merged.json',
+  mock = 'mockdata.json'
+}
+
+export const sources: Source[] =
+  import.meta.env.PROD || import.meta.env.USE_PROD ? [Source.v1, Source.v2] : [Source.mock]
```

Comment:
This isn't working for me. Also please update https://github.com/mozilla/dawg/blob/main/README.md#%EF%B8%8F-do-not-commit-these-files to reflect this very cool new change